release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1306,6 +1306,8 @@ To retain the catalog rebuilding functionality, use `--rebuild-catalog`. However
 
 * Previously, the addition of the `defaultChannel` field in the `mirror.operators.catalog.packages` file enabled users to specify their preferred channel, overriding the `defaultChannel` set in the operator. With this release, oc-mirror plugin now enforces an initial check if the `defaultChannel` field is set, users must also define it in the channels section of the `ImageSetConfig`. This update ensures that the specified `defaultChannel` is properly configured and applied during operator mirroring. (link:https://issues.redhat.com/browse/OCPBUGS-385[*OCPBUGS-385*])
 
+* Previously, when running a cluster with FIPS enabled, you might have received the following error when running the OpenShift CLI (`oc`) on a RHEL 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. With this release, the default OpenShift CLI (`oc`) is compiled with {op-system-base-full} 9 and works properly when running a cluster with FIPS enabled on {op-system-base} 9. Additionally, an OpenShift CLI (`oc`) compiled with {op-system-base} 8 is also provided, which must be used if you are running a cluster with FIPS enabled on {op-system-base} 8. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*], link:https://issues.redhat.com/browse/OCPBUGS-28540[*OCPBUGS-28540*])
```

Comment:
```suggestion
* Previously, when running a cluster with FIPS enabled, you might have received the following error when running the OpenShift CLI (`oc`) on a {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. With this release, the default OpenShift CLI (`oc`) is compiled with {op-system-base-full} 9 and works properly when running a cluster with FIPS enabled on {op-system-base} 9. Additionally, an OpenShift CLI (`oc`) compiled with {op-system-base} 8 is also provided, which must be used if you are running a cluster with FIPS enabled on {op-system-base} 8. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*], link:https://issues.redhat.com/browse/OCPBUGS-28540[*OCPBUGS-28540*])
```

Could also use `{oc-first}` if you like. 

I suppose the non-first instances should just be `oc`, but I can understand wanting to use a more normal noun phrase given the context ("the default `oc`" and "an `oc`" read p funny imo :sweat_smile:). Maybe "the default version of `oc`" and "a version of `oc`" would work? :thinking: So perhaps:

```suggestion
* Previously, when running a cluster with FIPS enabled, you might have received the following error when running the {oc-first} on a {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. With this release, the default version of `oc` is compiled with {op-system-base-full} 9 and works properly when running a cluster with FIPS enabled on {op-system-base} 9. Additionally, a version of `oc` compiled with {op-system-base} 8 is also provided, which must be used if you are running a cluster with FIPS enabled on {op-system-base} 8. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*], link:https://issues.redhat.com/browse/OCPBUGS-28540[*OCPBUGS-28540*])
```

Reply:
Good suggestions, thanks!